### ISSUE TITLE
NOT BLAZING FAST AND MEMORY SAFE 

### ISSUE THREAD

**Issue by Sanceilaks** [Daniel-Liu-c0deb0t/uwu#41]
https://github.com/Daniel-Liu-c0deb0t/uwu/blob/5016701694c431ccda5484bcb63380bed5db3db8/src/lib.rs#L33
WTF

**Comment by stopmotio** [comment]
owo whats this

**Comment by interdisziplina** [comment]
yeah..  this function n-needs to b-be unsafe a-as one c-couwd just pass a m-mutable wefewence to an empty s-swice s-safewy mmm.. which is instant u-ub

**Comment by stopmotio** [comment]
You guys are truly something else

**Comment by Sanceilaks** [comment]
still not safe
